openedx/core/djangoapps/course_apps/api.py

```diff
@@ -0,0 +1,63 @@
+"""
+Python APIs for Course Apps.
+"""
+from django.contrib.auth import get_user_model
+from opaque_keys.edx.keys import CourseKey
+
+from .models import CourseAppStatus
+from .plugin import CourseAppsPluginManager
+
+
+User = get_user_model()
+
+
+def is_course_app_enabled(course_key: CourseKey, app_id: str) -> bool:
+    """
+    Return if the app with the specified `app_id` is enabled for the
+    specified course.
+
+    Args:
+        course_key (CourseKey): Course key for course
+        app_id (str): The app id for a course app
+
+    Returns:
+        True or False depending on if the course app is enabled or not.
+    """
+    course_app = CourseAppsPluginManager.get_plugin(app_id)
+    is_enabled = course_app.default_enabled
```

Comment:
Why `default_enabled` interface is used when we have a class method for this case `is_enabled`?

Reply:
The `default_enabled` interface is to allow specifying whether the app should be automatically enabled for course runs where it's available or not. Otherwise, you will need to first mark it as available and then enable it. By setting this to true all new course runs will default to enabling the app, while old runs can still opt in.